nearcore/src/config.rs

```diff
@@ -1041,15 +1058,15 @@ impl LocalnetNodeParams {
 /// * `num_non_validators_archival` - Number of non-validator nodes to create and configure as an archival node (storing full chain history)
 /// * `num_non_validators_rpc` - Number of non-validator nodes to create and configure as an RPC node (eg. for sending transactions)
 /// * `num_non_validators` - Number of additional non-validator nodes to create
-/// * `tracked_shards` - Shards to track by all nodes, except for archival and RPC nodes which track all shards
+/// * `tracked_config` - Shards to track by all nodes, except for archival and RPC nodes which track all shards
```

Comment:
nit: tracked shards config for all nodes, except...

Reply:
Do you mean just renaming `tracked_config` to `tracked_shards_config`?

Reply:
both renaming and updating the comment, so the full line would be:
```
/// * `tracked_shards_config` - tracked shards config all nodes, except for archival and RPC nodes which track all shards
```